test/test_reactivities_bosch_hale.py

```diff
@@ -0,0 +1,66 @@
+import numpy.testing as npt
+import pytest
+
+## Testing DTn fusion reaction
+## Tested according to table VIII in H.-S. Bosch and G.M. Hale 1992 Nucl. Fusion 32 611
+@pytest.mark.parametrize(
+    "test, expected, precision",
+    [
+        (1.0, 6.857e-21, 21),
+        (2.0, 2.977e-19, 19),
+        (5.0, 1.366e-17, 18),
+        (10.0, 1.136e-16, 17),
+        (20.0, 4.330e-16, 16),
+    ])
+def test_DTn_reactivity(test, expected, precision):
+    import fusion_toolbox.reaction_rate.bosch_hale as bh
+    DTn_r= bh.Bosch_Hale_reactivity("DTn")
+    npt.assert_almost_equal(DTn_r.get_reactivity(test), expected, decimal=precision) 
+
+## Testing DDn fusion reaction
+## Tested according to table VIII in H.-S. Bosch and G.M. Hale 1992 Nucl. Fusion 32 611
+@pytest.mark.parametrize(
+    "test, expected, precision",
+    [
+        (1.0, 9.933e-23, 24),
+        (2.0, 3.110e-21, 22),
+        (5.0, 9.128e-20, 21),
+        (10.0, 6.023e-19, 20),
+        (20.0, 2.603e-18, 19),
+    ])
+def test_DDn_reactivity(test, expected, precision):
+    import fusion_toolbox.reaction_rate.bosch_hale as bh
+    DDn_r= bh.Bosch_Hale_reactivity("DDn")
+    npt.assert_almost_equal(DDn_r.get_reactivity(test), expected, decimal=precision) 
+
+## Testing DDp fusion reaction
+## Tested according to table VIII in H.-S. Bosch and G.M. Hale 1992 Nucl. Fusion 32 611
+@pytest.mark.parametrize(
+    "test, expected, precision",
+    [
+        (1.0, 1.017e-22, 23),
+        (2.0, 3.150e-21, 22),
+        (5.0, 9.024e-20, 21),
+        (10.0, 5.781e-19, 20),
+        (20.0, 2.399e-18, 20),
+    ])
+def test_DDp_reactivity(test, expected, precision):
+    import fusion_toolbox.reaction_rate.bosch_hale as bh
+    DDp_r= bh.Bosch_Hale_reactivity("DDp")
+    npt.assert_almost_equal(DDp_r.get_reactivity(test), expected, decimal=precision) 
+
+## Testing D3Hep fusion reaction
+## Tested according to table VIII in H.-S. Bosch and G.M. Hale 1992 Nucl. Fusion 32 611
+@pytest.mark.parametrize(
+    "test, expected, precision",
```

Comment:
Why do we need to parametrize the precision here?

Reply:
The Bosch Hale reactivities are just parameterizations fit to the fusion reactivity. So the values it gives will not exactly match the experimental or integrated reactivities. The precision is needed to make the tests pass when the values are close enough.